rosdep/python.yaml

```diff
@@ -3465,6 +3469,10 @@ python3-pep8:
   ubuntu: [python3-pep8]
 python3-setuptools:
   ubuntu: [python3-setuptools]
+wheel:
```

Comment:
Thanks. Unfortunately it's not well documented. I've added a few simple things here: https://github.com/ros/rosdistro/pull/15373